views/snippets/form_date.html

```diff
@@ -21,7 +21,7 @@ <h1 class="heading-small">What is your date of birth?</h1>
 
       <div class="form-group form-group-year">
         <label for="dob-year">Year</label>
-        <input class="form-control" id="dob-year" name="dob-year" type="number" pattern="[0-9]*" min="0" max="2014">
+        <input class="form-control" id="dob-year" name="dob-year" type="number" pattern="[0-9]*" min="1910" max="2014">
```

Comment:
Just wondering if keyboard users are more likely to use up/down or type in the number? Also we normally validate on 115 years old as that's the oldest person.

It'd be a real shame if we can't support 2 digit years as this is a common pattern for users.